src/components/common/ProductBox/ProductBox.module.scss

```diff
@@ -75,4 +77,14 @@
     justify-content: space-between;
     align-items: center;
   }
+
+  &:hover{
+    .buttons{
+      opacity: 1;
+    }
+
+    .price :first-child{
+      background-color: #d58e32;
```

Comment:
It would be great if you'd put this color into SCSS variable. Check file `settings.scss` in `styles` directory.